src/components/Search/SearchByNameResults.tsx

```diff
@@ -43,10 +43,11 @@ const EmptySearchByNameResult = () => {
 const SearchByNameResult = (props: { item: UDTQueryResult }) => {
   const { t } = useTranslation()
   const { item } = props
-  const { typeHash, symbol } = item
+  const { typeHash, fullName, symbol } = item
+  const displayName = fullName ?? symbol
```

Comment:
```suggestion
  const displayName = symbol ?? fullName
```

Symbol should be prior to the full name because an FT has a symbol and a full name while NFT only has a full name. If full name is prior to the symbol, symbol won't be used.